src/main/java/com/mosaic/Apiresponse.java

```diff
@@ -0,0 +1,33 @@
+package com.mosaic;
+
+import java.time.LocalDateTime;
+
+/**
+ * This is a class for deserializing future json responses from api to java-classes
+ * as api response will be List of forcasts ("intervals") I decided to make forcast itself an inner class of response
+ * example of response is in responseDummy.json under resources/json/.
+ * But im not exactly sure how to make a class which will make sense to deserialize such structure
+ *
+ * Implementation of deserializer with Jackson objectMapper to be done in next step (?)
+ *
+ */
+public class Apiresponse {
+
+
+    public class Weatherforecast {
+        private LocalDateTime startTime;
+        private float precipitationIntensity;
```

Comment:
I have a golden rule - I never use primitives in DTO (Data Transfer Objects).
The reason is simple - if there is no value specified - JVM initializes the field with the default one.
So with primitive type - there is no way to indicate the value is missing.

I would recommend using type Double here.